tests/RawProfilerHook/RawProfilerHook/CoRawProfilerHook.cpp

```diff
@@ -122,6 +167,54 @@ HRESULT STDMETHODCALLTYPE CCoRawProfilerHook::FunctionUnloadStarted(
 HRESULT STDMETHODCALLTYPE CCoRawProfilerHook::JITCompilationStarted(
     /* [in] */ FunctionID functionId,
     /* [in] */ BOOL fIsSafeToBlock){
+
+    ModuleID moduleId;
+    mdMethodDef methodToken;
+    HRESULT hr = S_OK;
+    IfFailRet(m_pRealProfilerInfo->GetFunctionInfo(functionId, nullptr, &moduleId, &methodToken));
+
+    if (moduleId == m_testModule)
+    {
+        if (m_firstJit)
+        {
```

Comment:
Extra tests:
- Double Alloc
- Order between IM and Raw maintained
- Order inversion with both modifying the method.